documentation/book/proc-creating-kafka-bridge-consumer.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-quickstart.adoc
+
+[id='proc-creating-kafka-bridge-consumer-{context}']
+= Creating a Kafka Bridge consumer
+
+Before you can perform any consumer operations in the Kafka cluster, you must first create a consumer by using the link:https://strimzi.io/docs/bridge/latest/#_createconsumer[consumers^] endpoint. The consumer is referred to as a __Kafka Bridge consumer__.
+
+.Procedure
+
+. Create a Kafka Bridge consumer in a new consumer group named `bridge-quickstart-consumer-group`: 
++
+[source,curl,subs=attributes+]
+----
+curl -X POST http://localhost:8080/consumers/bridge-quickstart-consumer-group \
+  -H 'content-type: application/vnd.kafka.v2+json' \
+  -d '{
+    "name": "bridge-quickstart-consumer",
+    "auto.offset.reset": "earliest",
+    "format": "json",
+    "enable.auto.commit": "false",
+    "fetch.min.bytes": "512",
+    "consumer.request.timeout.ms": "30000"
```

Comment:
even if it actually work for an issue in the OpenAPI validator, I would use the right boolean and number values for the last three properties to be compliant with the OpenAPI spec, so:

```json
"enable.auto.commit": false,
"fetch.min.bytes": 512,
"consumer.request.timeout.ms": 30000
```

Reply:
OK, that's done.